### PR TITLE
CI: add CI entrypoint

### PR DESCRIPTION
**Pull request by bercarug**
Added placeholder script which will be
responsible for triggering tests for each PR.

Signed-off-by: Gabriel Bercaru <bercarug@amazon.com>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by andraprs — DISMISSED** [review]

**Review by ddraga — DISMISSED** [review]

**Review by andraprs — APPROVED** [review]

File: run-nitro-enclaves-acm-tests [hunk 1]
```diff
@@ -167,7 +181,7 @@ function test_token_rsa_key {
 
 	out=$($P11NE_CLI describe-device)
 	ttl_init=$($P11NE_CLI describe-device \
-		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token")' \
+		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token-rsa")' \
```
Comment (ddraga) [inline run-nitro-enclaves-acm-tests:184]: Could you explain why was this change necessary?

File: run-nitro-enclaves-acm-tests [hunk 2]
```diff
@@ -273,7 +287,7 @@ function test_token_ec_secp384r1_key {
 
 	out=$($P11NE_CLI describe-device)
 	ttl_init=$($P11NE_CLI describe-device \
-		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token")' \
+		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token-secp384r1-ec")' \
```
Comment (ddraga) [inline run-nitro-enclaves-acm-tests:290]: Same question, why was this change necessary?

File: run-nitro-enclaves-acm-tests (hunk 1 above)
Comment (bercarug) [reply run-nitro-enclaves-acm-tests:184]: In the very first version of this test script, there was a single `test_token` function. Previous feedback suggested to split the test between two functions, one dealing with tokens initialized from rsa keys, while the second one using tokens initialized from secp3841 keys.

In the initial version, when there was a single function, the token label was called `test-token`, while the second version introduced `TOKEN_LABEL_RSA` and `TOKEN_LABEL_SECP384R1_EC`. These two variables were used in all places but these two jq select operations (which were two errors discovered during testing the pipeline when setting up the CI instance).

File: run-nitro-enclaves-acm-tests (hunk 2 above)
Comment (bercarug) [reply run-nitro-enclaves-acm-tests:290]: The same explanation as in the previous case applies here.

**Review by ddraga — APPROVED** [review]
